checksumpackage

```diff
@@ -114,6 +106,8 @@ _version_dfxml(){
 }
 
 while [ "${*}" != "" ] ; do
+    ABS_PATH=$(cd "$1"; pwd)
```

Comment:
Should add a check somewhere that "$1" is a directory. `if [ ! -d "$1" ] ; then echo OH NOOOOO ; fi`